<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeamSquirtleStyleConfig</class>
 <widget class="QWidget" name="TeamSquirtleStyleConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>257</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="1">
        <widget class="QLabel" name="_windowDragLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>W&amp;indows' drag mode:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>_windowDragMode</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="4">
        <widget class="QCheckBox" name="_viewDrawFocusIndicator">
         <property name="text">
          <string>Draw focus indicator in lists</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="_mnemonicsLabel">
         <property name="text">
          <string>&amp;Keyboard accelerators visibility:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>_mnemonicsMode</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QComboBox" name="_mnemonicsMode">
         <item>
          <property name="text">
           <string>Always Hide Keyboard Accelerators</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Show Keyboard Accelerators When Needed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Always Show Keyboard Accelerators</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="4">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>65</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QComboBox" name="_windowDragMode">
         <item>
          <property name="text">
           <string>Drag windows from titlebar only</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Drag windows from titlebar, menubar and toolbars</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Drag windows from all empty areas</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="1" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>49</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" colspan="4">
        <widget class="QCheckBox" name="_tabBarDrawCenteredTabs">
         <property name="text">
          <string>Center tabbar tabs</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QCheckBox" name="_toolBarDrawItemSeparator">
         <property name="text">
          <string>Draw toolbar item separators</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="4">
        <widget class="QCheckBox" name="_sliderDrawTickMarks">
         <property name="text">
          <string>Draw slider tick marks</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="4">
        <widget class="QCheckBox" name="_splitterProxyEnabled">
         <property name="text">
          <string>Enable extended resize handles</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Animations</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QLabel" name="_animationsDurationLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Anima&amp;tions duration:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>_animationsDuration</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QSpinBox" name="_animationsDuration">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string>ms</string>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QCheckBox" name="_animationsEnabled">
         <property name="text">
          <string>Enable animations</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="4">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Frames</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="_dockWidgetDrawFrame">
         <property name="text">
          <string>Draw frame around dockable panels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="_titleWidgetDrawFrame">
         <property name="text">
          <string>Draw frame around page titles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="_sidePanelDrawFrame">
         <property name="text">
          <string>Draw frame around side panels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="_menuItemDrawThinFocus">
         <property name="text">
          <string>Draw a thin line to indicate focus in menus and menubars</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>117</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Scrollbars</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="_scrollBarAddLineLabel">
         <property name="text">
          <string>Bottom arrow button t&amp;ype:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>_scrollBarAddLineButtons</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="_scrollBarSubLineLabel">
         <property name="text">
          <string>&amp;Top arrow button type:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>_scrollBarSubLineButtons</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="_scrollBarSubLineButtons">
         <item>
          <property name="text">
           <string>No Buttons</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>One Button</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Two Buttons</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="_scrollBarAddLineButtons">
         <item>
          <property name="text">
           <string>No Buttons</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>One Button</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Two Buttons</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>_tabBarDrawCenteredTabs</tabstop>
  <tabstop>_toolBarDrawItemSeparator</tabstop>
  <tabstop>_sliderDrawTickMarks</tabstop>
  <tabstop>_splitterProxyEnabled</tabstop>
  <tabstop>_mnemonicsMode</tabstop>
  <tabstop>_windowDragMode</tabstop>
  <tabstop>_animationsEnabled</tabstop>
  <tabstop>_animationsDuration</tabstop>
  <tabstop>_dockWidgetDrawFrame</tabstop>
  <tabstop>_titleWidgetDrawFrame</tabstop>
  <tabstop>_sidePanelDrawFrame</tabstop>
  <tabstop>_menuItemDrawThinFocus</tabstop>
  <tabstop>_scrollBarSubLineButtons</tabstop>
  <tabstop>_scrollBarAddLineButtons</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_animationsEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>_animationsDurationLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_animationsEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>_animationsDuration</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
